tests/fsharpqa/Source/Warnings/env.lst

```diff
@@ -2,22 +2,6 @@
 	SOURCE=FS0988AtEndOfFile.fs # FS0988AtEndOfFile.fs
 	SOURCE=EqualsInsteadOfInInForLoop.fs # EqualsInsteadOfInInForLoop.fs
 	SOURCE=DontWarnExternalFunctionAsUnused.fs SCFLAGS="--warnon:1182 --warnaserror+" # DontWarnExternalFunctionAsUnused.fs
-	SOURCE=SuggestTypesInModule.fs # SuggestTypesInModule.fs
-	SOURCE=SuggestGenericType.fs # SuggestGenericType.fs
-	SOURCE=SuggestTypeParameters.fs # SuggestTypeParameters.fs
-	SOURCE=SuggestNamespaces.fs # SuggestNamespaces.fs
-	SOURCE=SuggestModules.fs # SuggestModules.fs
-	SOURCE=SuggestMethods.fs # SuggestMethods.fs
-	SOURCE=SuggestAttributes.fs # SuggestAttributes.fs
-	SOURCE=SuggestUnionsInPatternMatch.fs SCFLAGS="--vserrors" # SuggestUnionsInPatternMatch.fs
-	SOURCE=SuggestUnionTypeForRQAUnions.fs SCFLAGS="--vserrors" # SuggestUnionTypeForRQAUnions.fs
-	SOURCE=SuggestRecordTypeForRQARecords.fs SCFLAGS="--vserrors" # SuggestRecordTypeForRQARecords.fs
-	SOURCE=SuggestRecordLabels.fs SCFLAGS="--vserrors" # SuggestRecordLabels.fs
-	SOURCE=SuggestUnionCases.fs SCFLAGS="--vserrors" # SuggestUnionCases.fs
-	SOURCE=SuggestArrayModuleFunctions.fs SCFLAGS="--vserrors" # SuggestArrayModuleFunctions.fs
-	SOURCE=SuggestTypesInNamespace.fs # SuggestTypesInNamespace.fs
-	SOURCE=SuggestTypesInNamespaceVS.fs SCFLAGS="--vserrors" # SuggestTypesInNamespaceVS.fs
-	SOURCE=SuggestAsyncModule.fs SCFLAGS="--vserrors" # SuggestAsyncModule.fs
 	SOURCE=SuggestDoubleBacktickIdentifiers.fs SCFLAGS="--vserrors" # SuggestDoubleBacktickIdentifiers.fs
```

Comment:
Counted 16, but this should be at parity with the deleted files: 21 (the tests should be 20 because 2 tests were identical with a different name)

Reply:
thx